opaque_keys/edx/tests/test_asset_locators.py

```diff
@@ -51,7 +51,7 @@ def test_deprecated_son(self, key_cls, prefix, tag, source):
         source_key = key_cls(*source, deprecated=True)
         son = source_key.to_deprecated_son(prefix=prefix, tag=tag)
         self.assertEqual(
-            son.keys(),
+            list(son.keys()),
```

Comment:
https://pymongo.readthedocs.io/en/4.4.0/migrate-to-pymongo4.html#son-items-now-returns-dict-items-object

This changes seems breaking and need to update at multiple places across edx.